Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -662,8 +701,12 @@ protected override void ProcessRecord()
     /// Replaces the contents of a Cloud Storage object with data from the local disk or a value
     /// from the pipeline.
     /// </para>
+    /// <example>
+    ///   <para>Update the contents of the Storage Object with the string "OK".</para>
+    ///   <para><code>PS C:\> "OK" | Write-GcsObject -Bucket "widget-co-logs" -ObjectName "status.txt"</code></para>
+    /// </example>
     /// </summary>
-    [Cmdlet(VerbsCommunications.Write, "GcsObject")]
+    [Cmdlet(VerbsCommunications.Write, "GcsObject"), OutputType(new string[] { })]
```

Comment:
I would argue for leaving off OutputType entirely when the cmdlet does not output anything, as it adds not external value.